.devcontainer/Dockerfile

```diff
@@ -93,7 +93,7 @@ COPY ./Makefile ./
 RUN make install-kind
 RUN make install-kubebuilder
 RUN make install-kustomize
-RUN make install-test-dependency
+# RUN make install-test-dependency
```

Comment:
is this change widely applicable or was it a local workaround?

Reply:
I think Janani/Justin confirmed it this target we removed/renamed. Maybe I should just remove it.

Reply:
I will try to change it to ```install-test-dependencies``` as Janani indicated.